Formula/opensearch-dashboards.rb

```diff
@@ -0,0 +1,72 @@
+class OpensearchDashboards < Formula
+  desc "Analytics and search dashboard for Elasticsearch"
+  homepage "https://opensearch.org/docs/dashboards/index/"
+  url "https://github.com/opensearch-project/OpenSearch-Dashboards.git",
+    tag:      "1.0.0",
+    revision: "b15ad258e6960003e609a70889cda5fb39f90947"
+  license "Apache-2.0"
+
+  depends_on "python@3.9" => :build
+  depends_on "yarn" => :build
+  depends_on "node@10" # Switch to `node` after https://github.com/opensearch-project/OpenSearch-Dashboards/issues/406
+
+  def install
+    inreplace "package.json", /"node": "10\.\d+\.\d+"/, %Q("node": "#{Formula["node@10"].version}")
+
+    system "yarn", "osd", "bootstrap"
+    system "node", "scripts/build", "--release", "--skip-os-packages", "--skip-archives"
+
+    cd "build/opensearch-dashboards-#{version}-darwin-x64" do
+      inreplace Dir["bin/*"],
+                "NODE=\"${DIR}/node/bin/node\"",
+                "NODE=\"#{Formula["node@10"].opt_bin/"node"}\""
+
+      inreplace "config/opensearch_dashboards.yml",
+                "/var/run/opensearchDashboards.pid",
+                var/"run/opensearchDashboards.pid"
+
+      (etc/"opensearch-dashboards").install Dir["config/*"]
+      rm_rf Dir["{config,data,node,plugins}"]
+
+      prefix.install Dir["*"]
+    end
+  end
+
+  def post_install
+    ln_s etc/"opensearch-dashboards", prefix/"config"
+    (prefix/"data").mkdir
+    (prefix/"plugins").mkdir
+  end
+
+  def caveats
+    <<~EOS
+      Config: #{etc}/opensearch-dashboards/
+      If you wish to preserve your plugins upon upgrade, make a copy of
+      #{opt_prefix}/plugins before upgrading, and copy it into the
+      new keg location after upgrading.
+    EOS
+  end
```

Comment:
I know you inherited this from `kibana` but I wonder if we can convince this to install the plugins into `var`, as with `opensearch`:
https://github.com/Homebrew/homebrew-core/blob/f52a2bf4dccbfbc22ec0ec2d0998a083c90e1ab1/Formula/opensearch.rb#L58-L76

Reply:
It looks like it's impossible to do 😞 
In all the places it's hardcoded to something like "<project root>/plugins". and it's not customisable. 
For example https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1.0.0/src/cli_plugin/install/settings.js#L67

Reply:
My understanding is that it's the same for `opensearch`, we just trick it with a symlink in order to make it put plug-ins in a static location.

Reply:
You're right!
I've updated the formula, now it's inlined with `opensearch` a bit more.